src/index.ts

```diff
@@ -11,7 +10,6 @@ import {renderStream} from './renderStream';
 import {PollingConfig} from './types';
 
 dotenv.config();
-Sentry.init({dsn: process.env.SENTRY_DSN});
```

Comment:
Damn, I guess we'll only have sentry for the server version huh 😬

Reply:
I don't fully understand how it works, but yeah I had to add the [express integration](https://github.com/getsentry/chartcuterie/pull/54/files#diff-4a3797637f93391dd49a34160aec29d8a0390c214ca2e2ef1ccdf284c35f6998R21) with the `app`

`new Integrations.Express({app})`

Reply:
I wonder if we can add the integration retroactively after the inital `Sentry.init`

Reply:
I don't know enough about node.js to quickly figure out if we can do this 😬  do you have any suggestions?

Reply:
https://forum.sentry.io/t/add-integration-after-init/5260/4 here we go